README.md

```diff
@@ -8,6 +8,7 @@ Google Summer Of Code Project under CCExtractor Development
 [![GPLv3 license](https://img.shields.io/badge/License-GPLv3-blue.svg)](https://github.com/ccextractor/Rekognition/blob/master/LICENSE)
 
 ---
+![bannerimage](https://github.com/CCExtractor/Rekognition/banner.png)
```

Comment:
image hasn’t been pushed

Reply:
create a folder named assets and move the image there
you will need to provide an exception to ignoring .png files in gitignore